crates/bifrost/src/loglet.rs

```diff
@@ -150,6 +153,17 @@ pub trait Loglet: Send + Sync {
     async fn seal(&self) -> Result<(), OperationError>;
 }
 
+#[derive(Default, Clone, Copy, PartialEq, Eq)]
+pub enum FindTailAttr {
+    /// Finds the durable tail of the loglet.
+    #[default]
+    Durable,
```

Comment:
I'd suggest we call this "ConsistentRead" all find-tail operations will find the durable tail, but Consistent performs a consistent-read instead.